### PR TITLE
Update LeagueDAO.json

### PR DESCRIPTION
**Pull request by xiaona423**
LeagueDAO - zh

### CODE REVIEW

**Review by DarkGhost7 — APPROVED** [review]
lgtm @Bee926 could you tl check?

File: data/protocols/1/LeagueDAO.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     },
     "zh": {
       "name": "League DAO",
-      "description": "League DAO is combining the best of DeFi protocols, play-to-earn gaming, and NFTs to create a web3 fantasy gaming reality where users can own and trade the players on their team and where on-chain scoring is based on real-world stats. Users can have a say in protocol governance with its native LEAG token."
+      "description": "League DAO 集合了各大 DeFi 协议的优点，它是玩中赚的游戏、是 web3 虚幻游戏现实的 NFT。用户可以真正拥有并交易自己球队的球员，链上评分都是基于真实世界的统计数据。League DAO 是由 LEAG 持有者管理的。"
```
Comment (Bee926) [inline data/protocols/1/LeagueDAO.json:40]: Change: "League DAO 集合了各大 DeFi 协议的优点，它是玩中赚的游戏、是 web3 虚幻游戏现实的 NFT“
to: “League DAO 集合了 DeFi 协议，玩中赚链游和 NFT 的优点来创建一个 web3 虚幻游戏”

**Review by DarkGhost7 — APPROVED** [review]
lgtm

**Comment by Bee926** [comment]
done, good to go

**Pull request merged**